drivers/SmartThings/zigbee-window-treatment/profiles/window-treatment-micronic.yml

```diff
@@ -0,0 +1,19 @@
+name: window-treatment-micronic
+components:
+- id: main
+  capabilities:
+  - id: windowShade
+    version: 1
+  - id: refresh
+    version: 1
```

Comment:
all zigbee profiles should include the `firmwareUpdate` capability in their profiles, for bookkeeping reasons